src/index.ts

```diff
@@ -34,7 +36,11 @@ export default class ConfusionMatrix {
    * @param {any} [options.sort]
    * @return {ConfusionMatrix} - Confusion matrix
    */
-  static fromLabels(actual, predicted, options = {}) {
+  static fromLabels(
+    actual: any[],
```

Comment:
since the algorithm uses strict equality, I don't think we should allow anything else than `boolean`, `number` or `string`